contracts/pm/mixins/MixinTicketBrokerCore.sol

```diff
@@ -1,15 +1,15 @@
-pragma solidity ^0.5.11;
-// solium-disable-next-line
-pragma experimental ABIEncoderV2;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.8;
 
-import "./interfaces/MReserve.sol";
-import "./interfaces/MTicketProcessor.sol";
+import "./MixinReserve.sol";
+import "./MixinTicketProcessor.sol";
 import "./interfaces/MTicketBrokerCore.sol";
-import "./interfaces/MContractRegistry.sol";
-import "openzeppelin-solidity/contracts/cryptography/ECDSA.sol";
-import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./MixinContractRegistry.sol";
 
-contract MixinTicketBrokerCore is MContractRegistry, MReserve, MTicketProcessor, MTicketBrokerCore {
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+abstract contract MixinTicketBrokerCore is MixinReserve, MixinTicketProcessor, MTicketBrokerCore {
```

Comment:
Thoughts on continuing to inherit from MTicketProcessor instead so this abstract contract doesn't need to be aware of the impl of the functions in MixinTicketProcessor and those implementations would just be mixed in via the MixinTicketProcessor mixin in TicketBroker? Mainly interested in whether we can minimize the diff here if this change is not needed.